init/systemd/gitlab-workhorse.service

```diff
@@ -1,25 +1,24 @@
 #####################################################
 #
-# GitLab version    : 8.2 - 8.x
-# Contributors      : bjorn-oivind
+# GitLab version    : 8.2 - 9.x
+# Contributors      : bjorn-oivind, Keruspe
 # Downloaded from   : https://gitlab.com/gitlab-org/gitlab-recipes/tree/master/init/systemd
 #
 ####################################################
 
 [Unit]
 Description=Gitlab Workhorse handles slow HTTP requests for Gitlab.
 Requires=gitlab-unicorn.service
-Wants=gitlab-unicorn.service
 After=gitlab-unicorn.service
 
 [Service]
 Type=forking
 User=git
-WorkingDirectory=/home/git/gitlab-workhorse
+WorkingDirectory=/home/git/gitlab
```

Comment:
When I had the problem about the secret being not found months ago (when I did the change locally to fix it) the documentation used to say that you need to specify the secret path only if workingdirectory is not /home/git/gitlab